src/stream/src/from_proto/batch_query.rs

```diff
@@ -38,18 +39,26 @@ impl ExecutorBuilder for BatchQueryExecutorBuilder {
             table_id: table_desc.table_id,
         };
 
-        let pk_descs = table_desc
+        let order_types = table_desc
             .order_key
             .iter()
-            .map(OrderedColumnDesc::from)
+            .map(|desc| OrderType::from_prost(&ProstOrderType::from_i32(desc.order_type).unwrap()))
             .collect_vec();
-        let order_types = pk_descs.iter().map(|desc| desc.order).collect_vec();
 
         let column_descs = table_desc
             .columns
             .iter()
             .map(ColumnDesc::from)
             .collect_vec();
+        // TODO: remove this
+        let pk_descs = table_desc
```

Comment:
Could you please open another issue for removing `common`'s `OrderedColumnDesc`?

Reply:
https://github.com/singularity-data/risingwave/issues/3203